python_docs_theme/static/pydoctheme.css

```diff
@@ -29,6 +29,11 @@ pre {
     color: inherit;
 }
 
+a {
+    text-decoration: underline 1px;
```

Comment:
```suggestion
    text-decoration: underline dotted 1px;
```
Have you tried this?

Reply:
<details>
<summary>Lots of screenshots</summary>


<img width="822" alt="image" src="https://github.com/python/python-docs-theme/assets/1324225/85040fe1-b541-46bf-a9ba-2d3a724be02e">

And if dotted we could reduce the `text-underline-offset` from 0.3em. Here's 0.2em:

<img width="819" alt="image" src="https://github.com/python/python-docs-theme/assets/1324225/e31ee876-9fad-4773-9e18-596f989944b7">

0.1em:

<img width="821" alt="image" src="https://github.com/python/python-docs-theme/assets/1324225/7982ee05-957d-4c02-829a-f4d62546e70c">

Unspecified/default=auto:

<img width="827" alt="image" src="https://github.com/python/python-docs-theme/assets/1324225/8286da0c-c8f1-4066-af04-b8642df15473">



</details>

Reply:
However, dotted underlines are sometimes reserved for "tooltips", as used on the PEPs:

* https://peps.python.org/
* https://github.com/python/peps/pull/2838

Another options is to have a lighter underline colour, like https://peps.python.org/ does. This does make the underline harder to see, and we need to be careful not to defeat the purpose of adding underlines in the first place.